components/cam/bld/namelist_files/use_cases/2000_scream_hr.xml

```diff
@@ -65,8 +65,12 @@
 <deep_scheme             >off</deep_scheme>
 
 <!-- Macrophysics/microphysics coupling -->
-<cld_macmic_num_steps    > 6</cld_macmic_num_steps>
```

Comment:
Removing this line likely leaves no default setting if someone were to add a new grid that isn't listed below. Is this intentional, or is this case handled by the code itself?

Reply:
Good point about leaving in a default value. I'll add it back in in the next commit.